thinc/compat.py

```diff
@@ -77,5 +77,21 @@
 except ImportError:  # pragma: no cover
     h5py = None
 
+try:
+    from thinc_apple_ops import AppleOps
```

Comment:
I think we were using indirect imports to avoid cycles. I think this could already cause a cycle in `thinc_big_endian_ops` (since it imports `thinc.api`). And importing `thinc_apple_ops` eagerly means that we can e.g. never use `compat` in `numpy_ops` in the future (since `thinc_apple_ops` imports `numpy_ops`).

Reply:
Hmm, good point. I'll try to think of a better way to accomplish this.